dirty.html

```diff
@@ -1,205 +1,204 @@
+          <li>URL, URI</li>
+          <li>Networking</li>                    
+          <li>SSH</li>
+          <li>Sessions</li>
+          <li>Authenticatilion</li>
+          <li>Authorization</li>
+          <li>Web Scraping</li>
+          <li>Cookies</li>
+          <li>HTTP APIs</li>
+          <li>RESTful APIs</li>
+          <li>HTTParty</li>                    
+          <li>Uploads</li>
+          <li>File I/O</li>
+          <li>Amazon S3 AWS</li>
+          <li>Sublime</li>
+          <li>Environment variables</li>               
+        </ul>
+      </div>
+
+      <div>
+        <ul>
+          <li>Version Control</li>
+          <li>Git and Github</li>
+          <li>Sinatra</li>                    
+          <li>Rails</li>
+          <li>MVC</li>
+          <li>Models</li>
+          <li>Views</li>
+          <li>Controllers</li>
+          <li>Routing</li>                    
+          <li>i18n</li>
+          <li>Helpers</li>
+          <li>Mailers</li>
+          <li>Serializers</li>
+          <li>Modules</li>
+          <li>Callbacks</li>
+          <li>Rake Tasks</li>
+          <li>Rails Console</li>
+        </ul>
+      </div>
+
+      <div>
+        <ul>
+          <li>ActiveRecord</li>
+          <li>Migrations</li>
+          <li>Validations</li>
+          <li>Relational DBs</li>
+          <li>SQL</li>
+          <li>Joins</li>
+          <li>Includes</li>
+          <li>MongoDB</li>
+          <li>Redis</li>
+          <li>Query string</li>
+          <li>Deploying</li>
+          <li>Heroku</li>
+          <li>Mandrill</li>
+          <li>Twilio</li>
+          <li>Exceptions</li>
+          <li>Exception Handling</li>
+          <li>Logs</li>
+        </ul>
+      </div>
+
+
+      <div>
+        <ul>
+          <li>Cancan</li>
+          <li>Carrierwave</li>
+          <li>Bcrypt</li>
+          <li>Devise</li>
+          <li>Sidekiq</li>
+          <li>Textacular</li>
+          <li>Factory Girl</li>
+          <li>High Voltage</li>
+          <li>Rspec</li>
+          <li>Nokogiri</li>
+          <li>Acts as Draggable</li>
+          <li>Acts as Taggable</li>
+          <li>Will Paginate</li>
+          <li>Nested Forms</li>
+          <li>Geocoder</li>
+          <li>Jekyll</li>
+          <li>Better Errors</li>
+        </ul>
+      </div>
+
+      <div>
+        <ul>
+          <li>Javascript</li>
+          <li>jQuery</li>
+          <li>Coffeescript</li>
+          <li>JSON</li>
+          <li>AJAX</li>
+          <li>Ember</li>
+          <li>Jasmine</li>
+          <li>Underscore.js</li>                    
+          <li>Templates</li>
+          <li>Responsive design</li>
+          <li>HTML</li>
+          <li>HAML</li>
+          <li>Handlebars</li>
+          <li>ERB</li>                    
+          <li>CSS</li>
+          <li>SCSS, SASS</li>
+          <li>Bootstrap</li>
+        </ul>
+      </div>
+    </section>
+
+    <!-- This is the final article -->
+    <section>
```

Comment:
Think also whether all the articles need a separate section or not in the future... seems like a lot of sections here I reckon #foodForThought